specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-04-01/managedClusters.json

```diff
@@ -1386,8 +1386,7 @@
       "properties": {
         "adminUsername": {
           "type": "string",
-          "description": "The administrator username to use for Windows VMs.",
-          "pattern": "^[a-zA-Z0-9]+([._]?[a-zA-Z0-9]+)*$"
+          "description": "The administrator username to use for Windows VMs."
```

Comment:
Suggest to update the description as that in compute model.

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json#L9443-L9450

Reply:
Though it's out of the scope this fix, as it helps us understand the properties, why not. Thanks